CHANGELOG.unreleased.md

```diff
@@ -31,4 +31,6 @@
 
 <!--packages-start-->
 
+- @xen-orchestra/web-core minor
```

Comment:
`@xen-orchestra/web` has also been changed. (since this is a very small change, patch is enough)
```suggestion
- @xen-orchestra/web patch
- @xen-orchestra/web-core minor
```